.github/workflows/continuous-e2e-single-vfn-test.yaml

```diff
@@ -0,0 +1,23 @@
+name: Continuous E2E Single VFN Test
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  workflow_dispatch:
+
+jobs:
+  # Test transaction processing throughput at a single VFN
+  run-forge-chaos:
+    uses: ./.github/workflows/run-forge.yaml
+    secrets: inherit
+    with:
+      FORGE_NAMESPACE: forge-chaos
```

Comment:
You need to use a separate namespace - just use `forge-continuous-e2e-single-vfn`